src/libraries/System.Private.Xml.Linq/tests/xNodeReader/ErrorConditions.cs

```diff
@@ -22,8 +22,18 @@ public void Variation1()
                     {
                         using (XmlReader r = n.CreateReader())
                         {
-                            TestLog.Compare(r[-100000], null, "Error");
-                            TestLog.Compare(r[-1], null, "Error");
+                            r.Read();
+
+                            try
+                            {
+                                TestLog.Compare(r[-100000], null, "Error");
```

Comment:
this will fail on the first one and never execute a second testcase, this problem should go away if you use Assert.Throws

Reply:
also this will likely not work correctly because it will accept both null and the exception, once you convert tests to regular xunit you can use SkipOnTargetFramework attribute to skip test on full framework

Reply:
What is the functional difference between this test and the ones I touched in CXMLReaderAttrTest? They obviously differ in how the reader is built, should replicate them both as regular XUnit cases? Has the "CXML" prefix any significance?

Reply:
I think the only difference is that one creates reader from XDocument and other from XElement, they probably could be collapsed. I don't believe cxml has any special meaning - perhaps the author just meant to check if namespaces work correctly

Reply:
I will move all cases from both GetAttribute and indexer test cases to a regular test then. Should the converted tests still be part of the outer loop? And [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] to all cases, yes?

Reply:
better if you make them inner loop (no `[OuterLoop]` attribute)